pkg/resource/supplier.go

```diff
@@ -1,10 +1,15 @@
 package resource
 
-// Resource Supplier supply the list of resource.Resource, its the front to retrieve remote resources
+// Supplier Resource Supplier supply the list of resource.Resource, its the front to retrieve remote resources
```

Comment:
?

Reply:
I'll revert this, my linter does not like when comment don't start with the object name as it does not respect godoc... https://blog.golang.org/godoc